src/Middleware/Localization/src/RequestLocalizationOptions.cs

```diff
@@ -20,13 +20,18 @@ public class RequestLocalizationOptions
     public RequestLocalizationOptions()
     {
         RequestCultureProviders = new List<IRequestCultureProvider>
-            {
-                new QueryStringRequestCultureProvider { Options = this },
-                new CookieRequestCultureProvider { Options = this },
-                new AcceptLanguageHeaderRequestCultureProvider { Options = this }
-            };
+        {
+            new QueryStringRequestCultureProvider { Options = this },
+            new CookieRequestCultureProvider { Options = this },
+            new AcceptLanguageHeaderRequestCultureProvider { Options = this }
+        };
     }
 
+    /// <summary>
+    /// Gets or sets whether to use use user-selected culture or default system culture. Defaults to <c>true</c>.
```

Comment:
```suggestion
    /// Configures <see cref="CultureInfo.UseUserOverride "/>. Defaults to <c>true</c>.
```

The "user" in this case is the server admin or app developer (Windows only), but people could easily consider the "user" to be the person making the request. It's better to just link to the `CultureInfo.UseUserOverride` docs.